deployment/install.sh

```diff
@@ -218,9 +236,19 @@ elif [ $AGENT_TYPE == 'opentsdb' ]; then
 		createCronMinute $REPORTING_INTERVAL "${COMMAND_REPORTING}" $TEMPCRON
 	fi
 elif [ $AGENT_TYPE == 'kafka-logs' ]; then
-    sudo /usr/bin/nohup $PYTHONPATH $INSIGHTAGENTDIR/kafka_logs/getlogs_kafka.py -d $INSIGHTAGENTDIR -w $SERVER_URL -l $CHUNK_LINES &>$INSIGHTAGENTDIR/log/kafka-logs.log &
+	MONITRCLOC=/etc/monit/monitrc
```

Comment:
Sometimes the monit conf file name is monit.conf for some operating systems. So please copy everything to that file also. Or maybe find a way to detect which control file monit is using and update it.

Reply:
i am okay.